modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/fragments/group/marketplace-base-fragments/fragments/get-app-button/index.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable no-undef */
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+const productID = fragmentElement
```

Comment:
Please, change the variable name to productId.
```suggestion
const productId = fragmentElement
```